blockchain/v0/reactor_test.go

```diff
@@ -258,6 +258,7 @@ func TestBcBlockRequestMessageValidateBasic(t *testing.T) {
 	}
 
 	for _, tc := range testCases {
+		tc := tc
```

Comment:
first time I am seeing such construct, that's for sure :)

Reply:
why do we need to do it?

Reply:
In Go, the `tc` variable in the above loop iterations is actually a single variable, so in many cases (like the above), using it can create potential concurrency problems..

Reply:
https://github.com/kyoh86/scopelint